### DISCUSSION TITLE
A few label ideas

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#4756] [Ideas]
- pio 5
- needs discussion
- dev tools
- popup

**Comment by Weredime** [comment]
we don't need prio 5, we have prio 4
needs discussion: maybe
dev tools: thats an addon
popup: yeah true

**Reply by Samq64** [reply]
> dev tools: thats an addon

What about things related to the dev tools extension?

**Comment by mxmou** [comment]
> * needs discussion

`status: pending` is currently used for that.

> * popup

Use either `scope: webpages` (for the popup itself) or `scope: addon` (for Scratch Messaging or cloud games).
